iothub/service/src/Jobs/Models/JobQueryOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Specifies the options associated with job queries.
+    /// </summary>
+    public class JobQueryOptions : QueryOptions
```

Comment:
Inherits from the base and adds 2 more optional values, so we go from 3 to 1 params.